app/helpers/date_helper.rb

```diff
@@ -0,0 +1,5 @@
+module DateHelper
+  def humanize_date(date)
```

Comment:
This is definitely going to come in useful. Down the line we could even have it be something like "X days ago," "Y weeks ago," etc.